opentelemetry/plugindefinition.yaml

```
@@ -4,16 +4,27 @@
 apiVersion: greenhouse.sap/v1alpha1
 kind: PluginDefinition
 metadata:
-    name: opentelemetry-operator
+    name: opentelemetry
 spec:
-    version: 0.2.6
-    description: opentelemetry-operator
+    version: 0.2.7
+    displayName: OpenTelemetry
+    description: Observability framework for instrumenting, generating, collecting, and exporting telemetry data such as traces, metrics, and logs.
     icon: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/opentelemetry-operator/logo.png
     helmChart:
         name: opentelemetry-operator
```

Doesn't this need to change too?

```suggestion
        name: opentelemetry
```

---

If yes, the please change the chart, too https://github.com/cloudoperators/greenhouse-extensions/blob/a751d89040a09b11ef5c66a4a71506bd8c085d3f/opentelemetry/chart/Chart.yaml#L6

.. and all the gh actions mentioning it.

---

we can drop this, it is to explain which upstream chart is used, so good from my side.